recipes/cucumber-tag-expressions/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "cucumber-tag-expressions" %}
+{% set version = "2.0.4" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cucumber-tag-expressions-{{ version }}.tar.gz
+  sha256: 72197b9330c023ce2643847fd6659c5c000f7f286f5b42cbcfd19adb3be92d30
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - enum34  # [py<34]
```

Comment:
Conda-forge does not build packages for py<34, so maybe we can drop this and make it noarch?

Reply:
I'll try